geth-utils/gethutil/mpt/witness/gen_witness_transactions_test.go

```diff
@@ -87,34 +97,59 @@ func TestNonHashedTransactionsStackTrieGetProof(t *testing.T) {
 	fmt.Println("===")
 }
 
+func TestStackTrieInsertion_1Tx(t *testing.T) {
```

Comment:
Perhaps would be good to add some check that the last proof element returned by `GetProof` is really the transaction for which proof we asked?

Reply:
fixed in [056fa7f](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1784/commits/056fa7f9be01929e7a28c1905d0aa90e0bf46741).